examples/src/test/kotlin/examples/cnn/dogscats/CatDogTestSuite.kt

```diff
@@ -5,10 +5,8 @@
 
 package examples.cnn.dogscats
 
-import org.junit.jupiter.api.Disabled
 import org.junit.jupiter.api.Test
 
-@Disabled
```

Comment:
No needs to remove Disabled annotation here. All these examples are too heavy and take a lot of time during usual checks.